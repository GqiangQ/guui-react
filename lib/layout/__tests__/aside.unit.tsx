import {configure} from 'enzyme';
import * as Adapter from 'enzyme-adapter-react-16';
import * as React from 'react';
import * as renderer from 'react-test-renderer';
import Aside from '../aside';

configure({adapter: new Adapter()});

describe(Aside.displayName, () => {
  it('DisplayName', () => {
    expect(Aside.displayName).toEqual('Aside');
  });
  describe('props: className', () => {
    it('= xxx', () => {
      const tree = renderer
        .create(<Aside className="xxx"></Aside>)
        .toJSON();
      expect(tree).toMatchSnapshot();
    });
  });
  describe('props: style', () => {
    it(`= {color: 'red'}`, () => {
      const tree = renderer
        .create(<Aside style={{color: 'red'}}></Aside>)
        .toJSON();
      expect(tree).toMatchSnapshot();
    });
  });
});
