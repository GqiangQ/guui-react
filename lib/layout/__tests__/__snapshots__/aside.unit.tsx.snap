// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Aside props: className = xxx 1`] = `
<div
  className="gu-aside xxx"
/>
`;

exports[`Aside props: style = {color: 'red'} 1`] = `
<div
  className="gu-aside "
  style={
    Object {
      "color": "red",
    }
  }
/>
`;
